learn/advanced/filtering_and_faceted_search.md

```diff
@@ -342,23 +342,13 @@ If you only want recent `Planet of the Apes` movies that weren't directed by `Ti
 release_date > 1577884550 AND (NOT director = "Tim Burton" AND director EXISTS)
 ```
 
-### Filtering with `_geoRadius`
+### Filtering by geographic location
 
-If your documents contain `_geo` data, you can use the `_geoRadius` built-in filter rule to filter results according to their geographic position.
-
-`_geoRadius` establishes a circular area based on a central point and a radius. Results beyond this area will be excluded from your search. This filter rule requires three parameters: `lat`, `lng` and `distance_in_meters`.
-
-```
-_geoRadius(lat, lng, distance_in_meters)
-```
-
-`lat` and `lng` must be floating point numbers indicating a geographic position. `distance_in_meters` must be an integer indicating the radius covered by the `_geoRadius` filter.
-
-When using a <a id="downloadRestaurants" href="/restaurants.json" download="restaurants.json"> dataset of restaurants</a> containing geopositioning data, we can filter our search so it only includes places within two kilometers of our location:
+If your documents contain `_geo` data, you can filter results according to their geographic position using one of our built-in geosearch filter rules:
```

Comment:
```suggestion
If your documents contain `_geo` data, you can filter results according to their geographic position using our built-in geosearch filter rules `_geoRadius` and `_geoBoundingBox` :
```

Reply:
I actually did this on purpose. It doesn't seem unlikely the devs will create new geosearch filters (for instance, `_geoPolygon` that lets users search in shapes more complex than rectangles and circles), so I'm trying to keep it open in pages like this one, where geosearch is not the main subject.